assets/js/modules/analytics/components/common/AccountSelect.js

```diff
@@ -29,14 +29,34 @@ import Data from 'googlesitekit-data';
 import ProgressBar from '../../../../components/progress-bar';
 import { Select, Option } from '../../../../material-components';
 import { STORE_NAME, ACCOUNT_CREATE } from '../../datastore/constants';
+import { STORE_NAME as MODULES_TAGMANAGER } from '../../../tagmanager/datastore/constants';
 import { trackEvent } from '../../../../util';
 const { useSelect, useDispatch } = Data;
 
 export default function AccountSelect() {
-	const accounts = useSelect( ( select ) => select( STORE_NAME ).getAccounts() );
-	const accountID = useSelect( ( select ) => select( STORE_NAME ).getAccountID() );
-	const hasExistingTag = useSelect( ( select ) => select( STORE_NAME ).hasExistingTag() );
-	const isDoingGetAccounts = useSelect( ( select ) => select( STORE_NAME ).isDoingGetAccounts() );
+	const {
+		accounts,
+		accountID,
+		hasExistingTag,
+		hasGTMPropertyID,
+		isDoingGetAccounts,
+	} = useSelect( ( select ) => {
```

Comment:
Based on our conversation earlier today, let's use individual `useSelect` calls for these since they're simple and don't depend on each other.

Reply:
Updated.